### DISCUSSION TITLE
How can I update my turtlebot4 to Humble Hawksbill?

### DISCUSSION THREAD

**Discussion by dtwind62** [turtlebot/turtlebot4#115] [General]
Are there instructions somewhere that describes how to update a turtlebot4 to Humble Hawksbill? I noticed on the create 3 website, there is a Humble firmware update. I didn't see an humble version of the  .img file on the link in the user manual. I'm quite new to robotics, I'm still not comfortable compiling from source code, should I just wait for an official .img file to be released?

**Comment by roni-kreinin** [comment]
@dtwind62 I am working on releasing the official image next week. I recommend waiting for that.

**Comment by dtwind62** [comment]
I'll wait for that, thanks for the he prompt reply

**Reply by roni-kreinin** [reply]
Humble has been released for the robot. See #119

**Comment by dtwind62** [comment]
Thanks a bunch, such a great job in the update. Nice new features added.
